src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -220,6 +219,7 @@
     <Compile Include="System\Drawing\SystemIcons.Windows.cs" />
     <Compile Include="System\Drawing\ToolboxBitmapAttribute.cs" />
     <Compile Include="System\Drawing\Text\PrivateFontCollection.Windows.cs" />
+    <Compile Include="System\Drawing\UnsafeNativeMethods.cs" />
     <Compile Include="misc\DbgUtil.cs" />
     <Compile Include="misc\DpiHelper.cs" />
     <Compile Include="misc\ExternDll.cs" />
```

Comment:
These is `ExternDll.Unix.cs` include for Unix builds that has a bunch of Win32 .dll names. Is it another potential problem?

Reply:
Interesting. Good catch. I'll remove the windows specific pinvokes names to see what breaks and which windows pinvokes we are including.